lib/hanami/repository.rb

```diff
@@ -208,16 +209,33 @@ module ClassMethods
       #   end
       #
       #   UserRepository.adapter = adapter
+      # @since 0.5.0
+      # @api private
       def adapter=(adapter)
         @adapter = adapter
       end
 
-      # @since 0.5.0
```

Comment:
@pascalbetz @joneslee85 Yes, class => instance methods should be updated with `@version x.x.x`.